### PR TITLE
applications: nrf5340_audio: Update controller to 3349

### PR DESCRIPTION
**Pull request by gWacey**
OCT-2531

Update the controller to version 3349.

### CODE REVIEW

File: applications/nrf5340_audio/tools/buildprog/nrf5340_audio_dk_devices.json [hunk 1]
```diff
@@ -12,6 +12,6 @@
  {
         "nrf5340_audio_dk_snr": 1000,
         "nrf5340_audio_dk_dev": "headset",
-        "channel": "right"
+         "channel": "right"
```
Comment (rick1082) [inline applications/nrf5340_audio/tools/buildprog/nrf5340_audio_dk_devices.json:15]: This file should be ignore

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 2]
```diff
@@ -9,6 +9,16 @@ Changelog
 
 Notable changes to this controller are documented in this file.
 
+Controller v3347
```
Comment (rick1082) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:12]: Should this be 3349?

**Review by rick1082 — CHANGES_REQUESTED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 3]
```diff
@@ -9,6 +9,16 @@ Changelog
...
+****************
+
+The following changes have been introduced in the v3347 of the controller:
+
+Changes
+=======
+
+* ...........
```
Comment (alexsven) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:20]: Was this forgotten?

**Review by alexsven — CHANGES_REQUESTED** [review]
Agree with comments from Rick, and changelog is completely missing.

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst (hunk 3 above)
Comment (gWacey) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:20]: No Kristoffer said to open the PR so I could get input on this. Is there a list of changes somewhere I can copy in here?

Comment (koffes) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:20]: We may need a hand from @erikrobstad here so we can go through the relevant issues for this release.

File: applications/nrf5340_audio/tools/buildprog/nrf5340_audio_dk_devices.json (hunk 1 above)
Comment (koffes) [reply applications/nrf5340_audio/tools/buildprog/nrf5340_audio_dk_devices.json:15]: `git update-index --assume-unchanged <file>`

**Review by rick1082 — APPROVED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 4]
```diff
@@ -9,6 +9,19 @@ Changelog
 
 Notable changes to this controller are documented in this file.
 
+Controller v3349
+****************
+
+The following changes have been introduced in the v3347 of the controller:
```
Comment (alexsven) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:15]: Still wrong number.
```suggestion
The following changes have been introduced in the v3349 of the controller:
```

**Review by alexsven — CHANGES_REQUESTED** [review]

**Review by rick1082 — APPROVED** [review]

**Review by koffes — APPROVED** [review]

**Review by alexsven — APPROVED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 5]
```diff
@@ -9,6 +9,19 @@ Changelog
...
+
+The following changes have been introduced in the v3349 of the controller:
+
+Changes
+=======
+
+* Fixed Direct Test Mode (DTM), which has been broken after version 3307
+* Fixed issue where connection parameter update could lead to disconnection
+* Fixed issue where "stream stopped" and "disconnected" events were not triggered
+* Fixed issue where controller stopped responding when a disconnection happened
```
Comment (greg-fer) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:23]:
```suggestion
Bug fixes
=========

* Fixed the Direct Test Mode (DTM), which was broken after version 3307.
* Fixed an issue where an update to the connection parameter could lead to a disconnection.
* Fixed an issue where the "stream stopped" and "disconnected" events would not be triggered.
* Fixed an issue where the controller would stop responding when a disconnection happened.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst (hunk 5 above)
Comment (gWacey) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:23]: Will keep Changes header for consistency with the previous comments

**Review by greg-fer — APPROVED** [review]

**Pull request merged**